packages/db/src/resources/bytecodes.ts

```diff
@@ -29,36 +30,9 @@ export const bytecodes: Definition<"bytecodes"> = {
     type Instruction {
       opcode: String!
       programCounter: Int!
-      meta: InstructionMeta
-      sourceRange: SourceRange
       pushData: Bytes
     }
 
-    type InstructionMeta {
```

Comment:
Oh man what old version of code-utils were you on when you wrote this :P

Reply:
Previously I thought the source range information belonged in bytecodes, but now I know that not to be the case. In the future, I figure we'll grab this sorta information and put it in `src/resources/compilations` underneath `SourceMap`

Reply:
Oh this comment was just a joke about `InstructionMeta` (not the source range stuff). :P  I notice that it includes information that `code-utils` used to return but doesn't anymore.  I've actually thought about putting back in the pushes and pops information, but not the rest, no way... those fields never really made a lot of sense...